### PR TITLE
homework4

### PR DESCRIPTION
**Pull request by OleksandrBochenko**

### CODE REVIEW

File: repeat.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+repeat_str = ('name')
```
Comment (MarinaZelenska) [inline repeat.py]: What is this repeat_str = ('name')? Fix it

File: repeat.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+multiply_str = 3
+result = repeat_str * multiply_str
```
Comment (MarinaZelenska) [inline repeat.py:3]: Please, use input() function

File: turn_over.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+turn_str = 'spirit'
```
Comment (MarinaZelenska) [inline turn_over.py]: Please, use input() function

**Review by MarinaZelenska — APPROVED** [review]

File: turn_over.py (hunk 3 above)
Comment (OleksandrBochenko) [reply turn_over.py]: Fixed

File: repeat.py (hunk 2 above)
Comment (OleksandrBochenko) [reply repeat.py:3]: Fixed

File: repeat.py (hunk 1 above)
Comment (OleksandrBochenko) [reply repeat.py]: Fixed

**Pull request merged**